src/room/GroupCallInspector.tsx

```diff
@@ -460,12 +466,19 @@ export function GroupCallInspector({
       className={styles.inspector}
     >
       <div className={styles.toolbar}>
-        <button onClick={() => setCurrentTab("sequence-diagrams")}>
-          Sequence Diagrams
-        </button>
-        <button onClick={() => setCurrentTab("inspector")}>Inspector</button>
+        {showCallFeedDebugInfo === true && (
```

Comment:
This is what I was talking about in standup, btw — call feed debug info isn't related to the sequence diagrams. I don't think there's any reason to have options to show/hide tabs in the debug inspector. I would just show all of these tabs unconditionally.